HW23/roles/hw23/handlers/main.yml

```diff
@@ -0,0 +1,27 @@
+- name: "Start Docker service"
```

Comment:
Generally handlers are used for restarting services. Enabling them should be a part of main code.
Same for nginx